### PR TITLE
Correctlyparse metadata

### PR DESCRIPTION
**Pull request by carolanitz**
### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md) and run `bundle exec fastlane lint` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### Description

This fixes the black screen when you play music files on Apple TV

### CODE REVIEW

File: Apple-TV/Playback/VLCFullscreenMovieTVViewController.m [hunk 1]
```diff
@@ -831,28 +832,25 @@ - (void)mediaPlayerStateChanged:(VLCMediaPlayerState)currentState
...
 
-- (void)displayMetadataForPlaybackController:(VLCPlaybackController *)controller
-                                       title:(NSString *)title
-                                     artwork:(UIImage *)artwork
-                                      artist:(NSString *)artist
-                                       album:(NSString *)album
-                                   audioOnly:(BOOL)audioOnly
+- (void)displayMetadataForPlaybackController:(VLCPlaybackController *)controller metadata:(VLCMetaData *)metadata
 {
+    NSLog(@"%@",metadata);
```
Comment (fkuehne) [inline Apple-TV/Playback/VLCFullscreenMovieTVViewController.m]: Consider using APLog here to not leak the info in production.

File: Apple-TV/Playback/VLCFullscreenMovieTVViewController.m [hunk 2]
```diff
@@ -831,28 +832,25 @@ - (void)mediaPlayerStateChanged:(VLCMediaPlayerState)currentState
...
     self.titleLabel.text = title;
-
-    if (audioOnly) {
+    if (metadata.isAudioOnly) {
         self.audioArtworkImageView.image = nil;
         self.audioDescriptionTextView.hidden = YES;
         [self stopAudioDescriptionAnimation];
 
-        if (artist != nil && album != nil) {
+        if (artist != nil && metadata.albumName != nil) {
```
Comment (fkuehne) [inline Apple-TV/Playback/VLCFullscreenMovieTVViewController.m]: consider caching the album name as you will call this property 3 times, so you could save 2 selector dispatches.

File: SharedSources/VLCMetadata.m [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
-//  VLCMediaPlayer + Metadata.m
-//  VLC
-//
-//  Created by Carola Nitz on 9/27/17.
-//  Copyright © 2017 VideoLAN. All rights reserved.
-//
+/*****************************************************************************
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2017 VideoLAN. All rights reserved.
```
Comment (fkuehne) [inline SharedSources/VLCMetadata.m]: Consider updating the copyright date.

File: SharedSources/VLCMetadata.m [hunk 4]
```diff
@@ -53,19 +63,54 @@ - (void)updateMetadataFromMediaPlayer:(VLCMediaPlayer *)mediaPlayer;
...
-            self.trackNumber = metaDict[VLCMetaInformationTrackNumber];
+    if (self.isAudioOnly) {
+        self.artworkImage = [VLCThumbnailsCache thumbnailForManagedObject:item];
+
+        if (self.artworkImage) {
+            if (self.artist)
+                self.title = [self.title stringByAppendingFormat:@" — %@", self.artist];
+            if (self.albumName)
+                self.title = [self.title stringByAppendingFormat:@" — %@", self.albumName];
         }
```
Comment (fkuehne) [inline SharedSources/VLCMetadata.m]: Consider caching the metadata variables to save 1 selector dispatch per property.

File: Apple-TV/Playback/VLCFullscreenMovieTVViewController.m (hunk 1 above)
Comment (carolanitz) [reply Apple-TV/Playback/VLCFullscreenMovieTVViewController.m]: ah that can even go away! totally missed deleting that

**Comment by carolanitz** [comment]
merged and pushed with 2848a7dd1959fb30b072eb6fbc6aac06ae9dc607 & 9b53ce8bcade2aaecc70eb52f515705c14bff49c

**Pull request closed without merging**